### DISCUSSION TITLE
Объект ответа | Node.js с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/nodejsdev.ru#20] [Announcements]
# Объект ответа | Node.js с примерами кода

Справочник web-разработчика с примерами кода - Node.js

https://nodejsdev.ru/doc/response-object/

**Comment by amir248** [comment]
И как это в реальной жизни применить? Пример можно?

**Reply by bndby** [reply]
примеры тут еще есть
https://nodejsdev.ru/expressjs4/hello-world/
